lib/api/element-commands/isVisible.js

```diff
@@ -52,7 +52,7 @@ class IsVisible extends BaseElementCommand {
     if (this.api.capabilities) {
       // FIXME: temporary fix to get safari working
       const {browserName} = this.api.capabilities;
```

Comment:
You could do this instead, to check for the version:
```js
const {browserName, browserVersion} = this.api.capabilities;
if (browserName.toLowerCase() === 'safari' && parseInt(browserVersion, 10) >= 12) {

```

Reply:
I'll test this with Browserstack, but I suspect that might have to be used in conjunction with checking `start_process`. I'll try it out and amend the pull request as necessary.

Reply:
Added browserVersion to the checks, but retained `start_process` check